### PR TITLE
Initial ORM implementation

### PR DESCRIPTION
**Pull request by pking-moj**
## Link to relevant ticket

[CRM457-](https://dsdmoj.atlassian.net/browse/CRM457-)

## How to manually test the feature

### CODE REVIEW

File: alembic/env.py [hunk 1]
```diff
@@ -57,23 +57,31 @@ def run_migrations_online() -> None:
...
-    with connectable.connect() as connection:
-        context.configure(connection=connection, target_metadata=target_metadata)
+    connectable = config.attributes.get("connection", None)
+
+    if connectable is None:
+        postgres_url = "postgresql+psycopg2://{}:{}@{}/{}".format(
+            os.getenv("POSTGRES_USERNAME", "test"),
+            os.getenv("POSTGRES_PASSWORD", "pass"),
+            os.getenv("POSTGRES_HOSTNAME", "localhost"),
+            os.getenv("POSTGRES_dbNAME", "laa_crime_application_store"),
```
Comment (dwhenry) [inline alembic/env.py:67]: lower case db?

File: tests/data/database_test.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import os
+
+from laa_crime_application_store_app.data.database import get_db
+
+
+def test_get_db_returns_valid_session():
+    os.environ["POSTGRES_NAME"] = "laa_crime_application_store_test"
```
Comment (dwhenry) [inline tests/data/database_test.py:7]: it feels like we have multiple approaches to the name setting.

Here we are hardcoding it, but in the migrations we appended to the existing name, I feel this is slightly confusing and would prefer a consistent approach if possible. :)

**Review by dwhenry — COMMENTED** [review]
some small comments

File: alembic/env.py (hunk 1 above)
Comment (pking-moj) [reply alembic/env.py:67]: fixed

File: tests/data/database_test.py (hunk 2 above)
Comment (pking-moj) [reply tests/data/database_test.py]: Have normalised how we do this

**Review by dwhenry — APPROVED** [review]

File: laa_crime_application_store_app/routers/application.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+@router.get("/application/{app_id}", response_model=App)
+async def ping(app_id: UUID | None = None, db: Session = Depends(get_db)):
+    logger.info("GETTING_APPLICATION", application_id=app_id)
+    application = db.query(Application).filter(Application.id == app_id).first()
+    if application is not None:
+        logger.info("APPLICATION_FOUND", application_id=app_id)
+        return App(**application.__dict__)
+
+    logger.info("APPLICATION_NOT_FOUND", application_id=app_id)
+    return Response(status_code=400)
```
Comment (ivanELEC) [inline laa_crime_application_store_app/routers/application.py:25]: is the name of the method for this endpoint still meant to be ping? Seems to be an almost complete endpoint for our get requests (obviously more will be added to get versions etc)

**Review by ivanELEC — APPROVED** [review]

**Pull request merged**